integrated_channels/sap_success_factors/transmitters/content_metadata.py

```diff
@@ -29,103 +24,18 @@ def __init__(self, enterprise_configuration, client=SAPSuccessFactorsAPIClient):
             client=client
         )
 
-    def transmit(self, create_payload, update_payload, delete_payload, content_updated_mapping, **kwargs):
-        """
-        Transmit content metadata items to the integrated channel.
+    def _transmit_action(self, content_metadata_item_map, client_method, action_name):
```

Comment:
we don't have action-specific transformation hooks - ie an object is serialized for the remote system the same way whether its being created, updated, or deleted. That is fine for now but, SAP needs to set the `status` to "delete", this override lets us add an `INACTIVE` status to the items we're deleting before they're passed to the real `_transmit_delete`